core/src/test/java/org/apache/iceberg/TestSnapshotSummary.java

```diff
@@ -78,7 +78,7 @@ public void testFileSizeSummary() {
 
   @TestTemplate
   public void testFileSizeSummaryWithDeletes() {
-    assumeThat(formatVersion).isGreaterThan(1);
+    assumeThat(formatVersion).isEqualTo(2);
```

Comment:
are you planning to update these tests eventually to run against format version 3?

Reply:
Not sure. There are new tests for DVs in this class. I think this one needs V2 position deletes, we simply didn't have the validation before. V2 position deletes are not allowed in V3 tables.